src/main/java/net/javadiscord/javabot2/Bot.java

```diff
@@ -46,6 +52,11 @@
 	 */
 	public static ScheduledExecutorService asyncPool;
 
+	/**
+	 * The message cache.
+	 */
+	public static ConcurrentHashMap<MessageAuthor, LinkedList<Message>> messageCache;
```

Comment:
Also, is this message cache an integral part of the bot that will need to be accessed in many different places? It doesn't seem like it, so it would be best if we keep it out of public static access.